crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/attribute.py

```diff
@@ -152,3 +152,20 @@
     f(111111111111111111111111111111111111111111111111111111111111111111111111111111111)
     + 1
 ).bit_length()
+
+
+# Regression test for https://github.com/astral-sh/ruff/issues/16151
+result = (
+   (await query_the_thing(mypy_doesnt_understand))  # type: ignore[x]
+   .foo()
+   .bar()
+)
+
+(
+    (
+        a # trailing end-of-line
+        # trailing own-line
+    ) # trailing closing parentheses
+    # dangling before dot
+    .b # trailing end-of-line
+)
```

Comment:
What's the significance of this test case? It seems to be working fine on `main`

Reply:
I mainly wanted to make sure that I don't break this case.